src/Util/Frame.php

```diff
@@ -49,7 +60,7 @@ public function getFile()
     }
 
     /**
-     * @return int|null
+     * @return integer|null
```

Comment:
I am not sure why you want to get it aligned with the code. The type name is integer, even if php uses `int` internally. I guess it is rather because integer is long.